cmake/SearchForStuff.cmake

```
@@ -165,8 +165,11 @@ if (PKG_CONFIG_FOUND)
   set(SimTK_INSTALL_DIR ${SimTK_INSTALL_PREFIX})
   #list(APPEND CMAKE_MODULE_PATH ${SimTK_INSTALL_PREFIX}/share/cmake)
   find_package(Simbody)
-  if (Simbody_FOUND)
-    message (STATUS "Looking for Simbody - found")
+  if (Simbody_FOUND)  
+    # When simbody is found but it static libs, we need to add it manually.
+    if ("${Simbody_LIBRARIES}" STREQUAL "" OR "${Simbody_LIBRARIES}" STREQUAL "Simbody_LIBRARIES-NOTFOUND")
+    set(Simbody_LIBRARIES ${Simbody_STATIC_LIBRARIES}),  
```

```suggestion
      set(Simbody_LIBRARIES ${Simbody_STATIC_LIBRARIES})
```

Please indent and remove spurious `,` and trailing whitespace, thanks.

---

My bad. Thank you for your comments and your fixes.